### PR TITLE
fix(am): add migration guide for AM 3.15.6

### PR DESCRIPTION
**Pull request by leleueri**
related to https://github.com/gravitee-io/issues/issues/7415
related to https://github.com/gravitee-io/issues/issues/7418

🚀 CI was able to deploy the build of this PR, so you can now try it directly [here](https://graviteedocs.blob.core.windows.net/am-fix-3-15-6/index.html)

### CODE REVIEW

File: pages/am/3.x/installation-guide/installation-guide-migration.adoc [hunk 1]
```diff
@@ -10,6 +10,10 @@ WARNING: If you plan to skip versions when you upgrade, ensure that you read the
 
 WARNING: Be sure to run scripts on the correct database since `gravitee-am` is not always the default database! Check your db name by running `show dbs;`
 
+== Upgrade to 3.15.6
+
+WARNING: For security reason, default values for duration of the email mfaChallenge code and the resetPassword token have been reduced to 5 minutes. If you want to restore previous values, please update the gravitee.yaml file for the management and gateway service for these two values "`user.resetPassword.token`" and "`user.mfaChallenge.token`".
```
Comment (farmborough) [inline pages/am/3.x/installation-guide/installation-guide-migration.adoc:15]: "for **the** duration"

File: pages/am/3.x/installation-guide/installation-guide-production-ready.adoc [hunk 2]
```diff
@@ -228,4 +228,23 @@ Perform the following steps on the AM Gateway component:
...
\ No newline at end of file
+. You can also consider updating the `sameSite` to link:https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie#syntax[Strict^]  and adapt the session timeout:
+
+== Step 7: Mitigate Cross Site Scripting (XSS) and Cross Site Framing
+
+The AM Gateway implementes link:https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy[Content-Security-Policy] and link:https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Frame-Options[X-Frame-Options].
+
+It is recommended to use these two mechanisms to have a better control on the resources the user agent is allowed to load for a given page.
+
+For example if you don't want to render a page in a `<iframe>` element, you can define the `X-Frame-Options` to `DENY` or use the `frame-ancestors` directive of CSP.
```
Comment (farmborough) [inline pages/am/3.x/installation-guide/installation-guide-production-ready.adoc:239]: "in **an** <iframe> element"

**Review by farmborough — APPROVED** [review]

**Review by Johngren — APPROVED** [review]
LGTM

**Review by Johngren — APPROVED** [review]
LGTM

**Review by Johngren — APPROVED** [review]
LGTM

**Pull request merged**